packages/devtools_app/lib/src/screens/vm_developer/vm_developer_tools_screen.dart

```diff
@@ -78,47 +79,67 @@ class VMDeveloperToolsScreenBody extends StatefulWidget {
 class _VMDeveloperToolsScreenState extends State<VMDeveloperToolsScreenBody>
     with
         AutoDisposeMixin,
+        AutomaticKeepAliveClientMixin<VMDeveloperToolsScreenBody>,
         ProvidedControllerMixin<VMDeveloperToolsController,
             VMDeveloperToolsScreenBody> {
+  int _selectedIndex = 0;
+  late List<Widget> _views;
+
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
     initController();
   }
 
+  void onDestinationSelected(int i) {
+    setState(() => _selectedIndex = i);
```

Comment:
calling setState has the same effect as wrapping the content in a ValueListenableBuilder (it rebuilds the whole widget). So logically, this change is a no-op.

Reply:
Reverted